c/extras/py_ftrl.cc

```diff
@@ -212,198 +328,247 @@ Returns
 )");
 
 
-void Ftrl::reset(const PKArgs&) {
-  fm->init_model();
+void PyFtrl::reset_model(const PKArgs&) {
+  ft->init_model();
+}
+
+
+PKArgs PyFtrl::Type::args_reset_params(0, 0, 0, false, false, {},
+"reset_params", R"(reset_params(self)
+--
+
+Reset FTRL parameters to default values.
+
+Parameters
+----------
+    None
+
+Returns
+----------
+    None
+)");
+
+
+void PyFtrl::reset_params(const PKArgs&) {
```

Comment:
Why is this method needed? I could just write `ft.params = Ftrl().params`.

Reply:
Just a convenience method.

Reply:
Convenience methods are good for frequently used operations. For example, we have `frame.head()`, which is a shortcut for `frame[:10, :]`. But I don't think resetting parameters to default is that useful to warrant a dedicated method.

Reply:
Frankly, I'm not sure about all the use cases for the moment... anyways, it doesn't seem to be a big deal to keep it or remove. What I guess is more important are the values we should use for the default parameters, or may be don't have default parameters at all... Right now I just used a common sense and experience gained through the tests to set them up.

Reply:
Well, there is a cost in properly documenting/testing every new method, which is why I was suggesting to remove it.
As for the default values of the parameters, don't they have something in the paper regarding which parameters they found optimal for one of their experiments?

Reply:
I will easily remove it if it makes other's life simpler :) 

Regarding the default parameters, this is the only thing they say:
`The optimal value of α can vary a fair bit  depending  on  the  features  and  dataset,  and
β =  1  is usually good enough; this simply ensures that early learning rates are not too high.`

So I believe there are no "magic" numbers that will work for all the cases. I will just keep it as they're now, and we will see.